cidc_api/models.py

```diff
@@ -624,6 +638,23 @@ def find_by_id_and_email(cls, id, email, session):
             return None
         return upload
 
+    @with_default_session
+    def ingestion_success(self, session, commit=False):
```

Comment:
```suggestion
    def ingestion_success(self, session: Session, commit: bool = False):
```

Reply:
I would also suggest review/unification of default values for commit across models methods.  
Seems that it's somewhat a hodgepodge.